test_server_test.go

```diff
@@ -0,0 +1,316 @@
+package gocql
```

Comment:
Moved this out because it is useful to more than just `conn_test.go`. For instance, in `session_unit_test.go`.